src/routes/homepage/Homepage.jsx

```diff
@@ -0,0 +1,42 @@
+import { Carousel } from "../../components";
+import { Banner } from "../../components";
+import { FeaturedCard, Footer } from "../../components";
+import haruki from "../../assets/images/haruki.jpeg";
+import fiveac from "../../assets/images/fiveac.jpeg";
+import "./homepage.css";
+
+const featuredList = [
```

Comment:
you can move data in some other files. this way your main files will be cleaner

Reply:
This is temporary and will be removed as soon as I create by custom API with a featured endpoint.